plugins.json

```
@@ -93,17 +93,23 @@
     "name": "Get Environment Variables",
     "package": "netlify-plugin-get-env-vars",
     "repo": "https://github.com/shortdiv/netlify-plugin-get-env-vars",
-    "version": "1.0.0",
-    "status": "DEACTIVATED"
+    "version": "1.0.0"
```

```suggestion
    "version": "1.0.1"
```

Also, it looks like the latest version in npm is `1.0.1` but the `package.json` in the repository is `1.0.0`.